### ISSUE TITLE
Filter Make/Model Better

### ISSUE THREAD

**Issue by kjlaw89** [openalpr/openalpr#367]
I ran a sample image through the demo page and got back conflicting make make/model results. In cases where you have the make figured out, the model should be filtered from there.

Result:
![makemodel_detection](https://cloud.githubusercontent.com/assets/4183007/16009636/9c902da0-314b-11e6-9043-90f1aa7018aa.png)

You can see that the make returned back was a Cadillac, which is right, but the Make/Model was a GMC Sierra. 

Original:
![1-7-1465770302](https://cloud.githubusercontent.com/assets/4183007/16009645/a17f8ed2-314b-11e6-8731-e634ed74e861.jpg)

Here's a different angle where it gets (almost) everything right:

![right](https://cloud.githubusercontent.com/assets/4183007/16009844/6fe67524-314c-11e6-9cf7-cc0df5b495b4.png)

**Comment by matthill** [comment]
Yes, there may be ways to get aggregate results for these types of detections.  Right now, the make detection does not share any information from make/model.

Capturing from multiple angles and averaging the results should improve accuracy as well.
